awx/api/serializers.py

```diff
@@ -3028,6 +3025,7 @@ def get_related(self, obj):
             labels = self.reverse('api:job_template_label_list', kwargs={'pk': obj.pk}),
             object_roles = self.reverse('api:job_template_object_roles_list', kwargs={'pk': obj.pk}),
             instance_groups = self.reverse('api:job_template_instance_groups_list', kwargs={'pk': obj.pk}),
+            sharded_jobs = self.reverse('api:job_template_sharded_jobs_list', kwargs={'pk': obj.pk}),
```

Comment:
split here, too

Reply:
@dsesami could you update your review? the renaming to "slice" is now completed.